### ISSUE TITLE
Error in XQuestScores on Windows

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#2644]
Shadowed in the nightlies by the general C++11 failure.

```
3>..\..\..\OpenMS\src\openms\source\ANALYSIS\XLMS\XQuestScores.cpp(109): error C2668: 'pow' : ambiguous call to overloaded function
3>          c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\math.h(497): could be 'double pow(double,int)'
3>          c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\math.h(122): or       'double pow(double,double)'
3>          while trying to match the argument list '(double, OpenMS::Size)'
3>..\..\..\OpenMS\src\openms\source\ANALYSIS\XLMS\XQuestScores.cpp(113): error C2668: 'pow' : ambiguous call to overloaded function
3>          c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\math.h(497): could be 'double pow(double,int)'
3>          c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\math.h(122): or       'double pow(double,double)'
3>          while trying to match the argument list '(double, OpenMS::Size)'
```

**Comment by jpfeuffer** [comment]
So this is solved right? Please reopen if not.

**Comment by enetz** [comment]
This should be solved by #2650 , but I could not reproduce the error locally, so I could not test it directly.
